doc/getting_started/index.rst

```diff
@@ -80,29 +73,15 @@ Next, clone the Zephyr source code repositories from GitHub using the
    west update
 
 .. note::
-   You can replace :file:`zephyrproject` with the folder name of your choice.
-   West will create the named folder if it doesn't already exist.
-   If no folder name is specified, west initializes the current
-   working directory.
-
-.. note::
-   If you had previously cloned the zephyr repository manually using Git,
```

Comment:
I disagree with removing `-l` here. I think it has a lot of value, not only for CI systems that come with their own Git client that will pull the repo CI is supposed to run on, but also for users that might want to clone manually.

Reply:
I think that setting up CI is an advanced use case not suitable for the getting started guide, as is cloning the multiple repositories we require manually. (Cloning by hand is also already served by https://docs.zephyrproject.org/latest/guides/west/without-west.html#no-west, which is linked to above.)

Documenting `west init -l` was all well and good while we were converting people to west, but are these use cases really significant enough to document here? The west multirepo page does document this option clearly in the "Main Commands" section.

Reply:
Fair enough, you've both convinced me. I am always reluctant to remove information, it takes real courage to do so.